model/Storage.inc.php

```diff
@@ -236,7 +237,7 @@ public static function queueUpload($userID, Zotero_StorageFileInfo $info) {
 				$info->charset
 			)
 		);
-		
+
```

Comment:
Also, I tried to understand when should I put new lines around block and when not. Dan, do you have a simple rule how you decided when to put additional line and when not?